Test/test2/Rlimitouts0.bpl.expect

```diff
@@ -0,0 +1,21 @@
+Prover error: line 105 column 10: max. resource limit exceeded
+Rlimitouts0.bpl(9,16): Verification inconclusive (TestTimeouts0)
```

Comment:
I'm not sure sure if your code is responsible for this but `(TestTimeouts0)` looks badly formatted.